packages/client/src/debug/DebugCheckersTable.tsx

```diff
@@ -38,24 +31,7 @@ const returnedData = (apiTopic: topicProps) => {
   return (
     <tr key={title}>
       <td>
-        {imtrTopic && !imtrTopic.hasIMTR ? (
```

Comment:
Are you sure? Let's discuss. I put this here thoughtfully.

Reply:
It was there for a very good reason: "do the IMTR/api topic files really exist even though manually configured to be something else, eg: wrong `hasImtr` or a topic programmed as a OLO flow or Redirect?"

But now i've changed this page to work like this:
- show all api topics that are configured as permit checks: `apiTopic.filter((.) => isPermitCheck(...)` (mind it loops over `api topics`)
- show all OLO flow topics: `topics.filter((.) => !hasIMTR(...) ...` (mind it loops over `topics` from the client)
- show all permit forms `apiTopic.filter((.) => isPermitForm(...)` (mind it loops over `api topics`)
- show all the `api topics` that are not programmed in the client: `apiTopics.map((.) => apiTopic.filter((.) => !findClientTopic(...))`

I would say this already has security enough to not be able to have a topic which is over- or underconfigured. But please try to break it (-:

Reply:
As discussed offline